servers/xr_server.cpp

```diff
@@ -216,17 +216,12 @@ Ref<XRInterface> XRServer::get_interface(int p_index) const {
 };
 
 Ref<XRInterface> XRServer::find_interface(const String &p_name) const {
-	int idx = -1;
 	for (int i = 0; i < interfaces.size(); i++) {
 		if (interfaces[i]->get_name() == p_name) {
-			idx = i;
-			break;
+			return interfaces[i];
 		};
 	};
-
-	ERR_FAIL_COND_V_MSG(idx == -1, nullptr, "Interface not found.");
-
-	return interfaces[idx];
+	return nullptr;
```

Comment:
This compiled, so it's probably fine, but don't we usually do this like:

```suggestion
	return Ref<XRInterface>();
```

... for functions that return refcounted types?

Ah, and I guess the old code was basically doing that too. So this must fine, I just didn't realize you could do that :-)

Reply:
I guess it's just a cosmetic difference, so I'm good with whatever is the suggested style.

Reply:
Yeah we usually return a `Ref<T>()` explicitly.
I think `nullptr` works because it will be converted to a `Ref<T>` implicitly, but it's good to be explicit.